ducktape/cluster/windows_remoteaccount.py

```diff
@@ -96,10 +96,7 @@ def winrm_client(self):
 
         return self._winrm_client
 
-    def fetch_externally_routable_ip(self, is_aws):
-        if not is_aws:
-            raise NotImplementedError("Windows is only supported in AWS.")
-
+    def fetch_externally_routable_ip(self, is_aws=None):
```

Comment:
So do we support windows in aws now?

Reply:
in the past we only supported windows in aws for vagrant clusters, whats not made clear is this code path is only executed from the vagrant cluster file, so its not a great representation of whats actually supported.  This pr makes things a bit more generic.